src/Rings/MPolyMap/MPolyRing.jl

```diff
@@ -53,8 +53,8 @@ julia> R, (x, y) = polynomial_ring(K, ["x", "y"]);
 
 julia> F = hom(R, R, z -> z^2, [y, x])
 Ring homomorphism
-  from multivariate polynomial ring in 2 variables over GF(2, 2)
-  to multivariate polynomial ring in 2 variables over GF(2, 2)
+  from multivariate polynomial ring in 2 variables over K
+  to multivariate polynomial ring in 2 variables over K
```

Comment:
This is one case where I don't like the name printing. Could/should we make an exception for finite fields (as we already do for the singletons like `QQ`)?

Reply:
I don't see how this "drastically reduces information" -- K was defined a few lines above so this seems quite clear to me?